blockchain/state_transition.go

```diff
@@ -272,7 +285,8 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, kerr kerr
 
 	// Defer transferring Tx fee when DeferredTxFee is true
 	if st.evm.ChainConfig().Governance == nil || !st.evm.ChainConfig().Governance.DeferredTxFee() {
-		st.state.AddBalance(st.evm.Coinbase, new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), st.gasPrice))
```

Comment:
In `NewStateTransition`, we set gasPrice with effectiveGasPrice, so i think we can just use `st.gasPrice` in here.

Reply:
Please don't mind this comment